src/transformers/convert_graph_to_onnx.py

```diff
@@ -30,16 +30,66 @@ class OnnxConverterArgumentParser(ArgumentParser):
     def __init__(self):
         super(OnnxConverterArgumentParser, self).__init__("ONNX Converter")
 
-        self.add_argument("--pipeline", type=str, choices=SUPPORTED_PIPELINES, default="feature-extraction")
-        self.add_argument("--model", type=str, required=True, help="Model's id or path (ex: bert-base-cased)")
+        self.add_argument(
+            "--pipeline", type=str, choices=SUPPORTED_PIPELINES, default="feature-extraction",
+        )
+        self.add_argument(
+            "--model", type=str, required=True, help="Model's id or path (ex: bert-base-cased)",
+        )
         self.add_argument("--tokenizer", type=str, help="Tokenizer's id or path (ex: bert-base-cased)")
-        self.add_argument("--framework", type=str, choices=["pt", "tf"], help="Framework for loading the model")
+        self.add_argument(
+            "--framework", type=str, choices=["pt", "tf"], help="Framework for loading the model",
+        )
         self.add_argument("--opset", type=int, default=11, help="ONNX opset to use")
-        self.add_argument("--check-loading", action="store_true", help="Check ONNX is able to load the model")
-        self.add_argument("--use-external-format", action="store_true", help="Allow exporting model >= than 2Gb")
+        self.add_argument(
+            "--check-loading", action="store_true", help="Check ONNX is able to load the model",
+        )
+        self.add_argument(
+            "--use-external-format", action="store_true", help="Allow exporting model >= than 2Gb",
+        )
+        self.add_argument(
+            "--quantize", action="store_true", help="Quantize the neural network to be run with int8",
+        )
         self.add_argument("output")
 
 
+def generate_identified_filename(filename: Path, identifier: str) -> Path:
+    """
+    Append a string-identifier at the end (before the extension,  if any) to the provided filepath.
+    Args:
+        filename: pathlib.Path The actual path object we would like to add an identifier suffix
+        identifier: The suffix to add
+
+    Returns: String with concatenated indentifier at the end of the filename
+    """
+    return filename.parent.joinpath(filename.stem + identifier).with_suffix(filename.suffix)
+
+
+def ensure_onnxruntime_installed():
+    """
+    Check onnxruntime is installed and if the installed version match is recent enough.
+    Raises:
+        ImportError: If onnxruntime is not installed or too old version is found
+    """
+    try:
+        import onnxruntime
+
+        # We require 1.4.0 minimum
+        if int(onnxruntime.__version__.split(".")[1]) < 4:
```

Comment:
For example, version 2.0 cannot pass this check. How about using packaging.version to compare version.

Reply:
Thanks for pointing out @tianleiwu, that's one of package (_packaging.version_) I'm not super used to play with so didn't investigate much more. Will have a look 👍